### PR TITLE
Test combination of fault tolerance with @Transactional

### PR DESCRIPTION
**Pull request by jonhawkes**

### CODE REVIEW

**Comment by jonhawkes** [comment]
fixes #1053 

#build

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/tests/TxRetryTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    @Test
+    public void testRetry() throws Exception {
+        WebBrowser browser = createWebBrowserForTestCase();
+        getSharedServer().verifyResponse(browser, "/TxFaultTolerance/retry?testMethod=testRetry",
+                                         "SUCCESS");
+    }
+
+    @Test
+    public void testRetry2() throws Exception {
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/tests/TxRetryTest.java]: Please could you rename the tests to something more descriptive... something like testRetrySingleTran and testRetryMultiTran ??

Comment (jonhawkes) [reply dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/tests/TxRetryTest.java]: done

#build

**Comment by ewhoch** [comment]
Please give PRs a more sensible title and description.  The commits are also very vague.

**Comment by ewhoch** [comment]
Please fix the copyright change, too. It's not correct to change the only year from 2017 -> 2018 for an existing file.

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/bnd.bnd [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2017 IBM Corporation and others.
+# Copyright (c) 2018 IBM Corporation and others.
```
Comment (ewhoch) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/bnd.bnd]: This isn't correct. Unless the file is entirely rewritten, the format is "initial year, modified year" at most. Even just "initial year" is more correct than removing the initial year.

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/suite/FATSuite.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2018 IBM Corporation and others.
```
Comment (ewhoch) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/suite/FATSuite.java]: same here.

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Pull request merged**